src/index.spec.ts

```diff
@@ -24,8 +26,9 @@ describe("friendship-blaster", () => {
 
   const shutdownFblaster = async (): Promise<void> => {
     if (fblasterProcess) {
-      await fblasterProcess.shutdown();
+      const procCopy = fblasterProcess;
       fblasterProcess = undefined;
+      await procCopy.shutdown();
```

Comment:
Is there any difference?

Reply:
`shutdown` could be called twice before, because `fblasterProcess` remained defined while the `await` was happening.

Reply:
see the `if (fblasterProcess)` on line 28 (formerly line 26).

Reply:
OK I get it.